src/components/Calculator.js

```diff
@@ -7,10 +7,14 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+  const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
+
+  const onButtonClick = (val) => setValue(operators.includes(val) ? '' : val)
+
+  // function onButtonClick(val) {
```

Comment:
Could we not get rid of the commented out code?

Reply:
Oh whoops! I thought i did get rid of it